pkg/v1/tkg/kind/client.go

```diff
@@ -352,7 +352,14 @@ func (k *KindClusterProxy) getKindNetworkingConfig() kindv1.Networking {
 		default:
 			podSubnet = constants.DefaultIPv4ClusterCIDR
 		}
+	} else if ipFamily == constants.DualStackPrimaryIPv6Family {
+		// We expect the IPv4,IPv6 order for the CIDR in both cases for KinD until
+		// we bump to a version of the kind library that includes a fix for this issue:
+		// https://github.com/kubernetes-sigs/kind/issues/2484
+		podSubnets := strings.Split(podSubnet, ",")
+		podSubnet = fmt.Sprintf("%s,%s", podSubnets[1], podSubnets[0])
```

Comment:
Please factor out `podSubnet` calculation into its own method:
```go
podSubnet := k.podSubnet(ipFamily)
```

In that method, please avoid using `else if` (e.g. by returning from the `if` body).

Reply:
Sounds good. We ended up reading the ipFamily within the podSubnet/serviceSubnet functions and now the `getKindNetworkingConfig` function is very simple.